app/objects/c_schedule.py

```diff
@@ -13,9 +14,14 @@ class Meta:
         unknown = ma.EXCLUDE
 
     id = ma.fields.String()
-    schedule = ma.fields.Time(required=True)
+    schedule = ma.fields.String(required=True, metadata={"example": "5 4 * * *"})
```

Comment:
this generates a tiny merge conflict, but should be fine to add

Reply:
Yes, I have rebased the branch on master. You should not have any merge conflicts anymore.